index.html

```diff
@@ -0,0 +1,101 @@
+            </svg>    
+        </a>
+        <a href="#">
+            <svg width="15" height="20" viewBox="0 0 15 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M6.96416 20C11.6608 20 14.5833 16.5392 14.5833 12.0742C14.5833 10.3908 14.3092 8.87083 12.3983 8.41417C12.8633 7.14083 14.205 3.4575 14.205 2.2775C14.205 1.23833 13.58 0.3675 12.4658 0.3675C10.69 0.3675 9.09833 5.51167 8.59999 6.73833C8.21416 5.625 6.4075 0 4.81499 0C3.57749 0 2.99666 0.894167 2.99666 2.01917C2.99666 3.39833 4.35749 6.9725 4.85083 8.3475C4.60083 8.2575 4.32999 8.17917 4.05416 8.17917C3.12249 8.17917 2.13166 9.31583 2.13166 10.2342C2.13166 10.5817 2.32666 11.07 2.44999 11.3942C0.981661 11.785 0.416661 12.7458 0.416661 14.195C0.412495 17.015 3.13583 20 6.96416 20ZM10.9892 4.21917C11.1492 3.73833 11.8292 1.7075 12.4658 1.7075C12.8083 1.7075 12.9 2.055 12.9 2.3325C12.9 3.07833 11.3642 7.2 11.025 8.11333L9.67166 7.87917L10.9892 4.21917ZM4.24666 1.8875V1.88667C4.24666 1.42167 4.82416 0.101667 6.08916 3.72667L7.46666 7.645C6.84583 7.59417 6.36416 7.5275 6.05833 7.7C5.62416 6.575 4.24666 3.02417 4.24666 1.8875ZM4.15499 9.53083L4.15583 9.53167C5.32166 9.53167 6.82666 13.2267 6.82666 13.7267C6.82666 13.9258 6.63166 14.1717 6.40499 14.1717C5.57249 14.1717 3.34333 11.1675 3.34333 10.355C3.34666 10.0542 3.84833 9.53083 4.15499 9.53083ZM2.81499 12.5892C3.26833 12.5892 4.97583 14.945 4.97583 15.445C4.97583 15.6367 4.66916 15.7692 4.51083 15.7692C3.86999 15.7692 3.61916 15.1633 2.47749 13.7617C1.29499 14.9217 3.29333 17.1558 4.79833 17.1558C5.83666 17.1558 6.51333 16.2108 6.31083 15.515C6.45833 15.515 6.64083 15.5267 6.77666 15.4917C6.82083 16.55 7.13916 17.8117 8.43666 17.9017C8.43666 17.8667 8.51666 17.6242 8.51666 17.6125C8.51666 16.9325 8.09499 16.3392 8.09499 15.6475C8.09499 14.5417 8.95833 13.4717 9.83416 12.8467C10.1525 12.6125 10.5392 12.4675 10.9125 12.335C11.2992 12.1908 11.7092 12.0225 12.0025 11.7333C11.9583 11.2958 11.7758 10.9092 11.33 10.9092C10.2275 10.9092 6.53 11.0658 6.53 9.35833C6.53 9.09667 6.53416 8.84667 7.22333 8.84667C8.50916 8.84667 11.7733 9.15917 12.7283 9.98333C13.4483 10.6125 13.6958 14.405 11.4942 16.8075L11.4917 16.8083C10.3333 18.0583 8.8525 18.7067 7.125 18.7067C4.76083 18.7067 2.89333 17.4333 1.99416 15.2575C1.31333 13.5625 2.14583 12.5892 2.81499 12.5892ZM7.56583 11.8858C7.95333 12.0075 8.35083 12.0433 8.74916 12.1208C8.45416 12.3317 8.1925 12.59 7.94083 12.8667C7.82916 12.535 7.69333 12.2108 7.56583 11.8858Z" fill="#505F79"/>
+            </svg>                
+        </a>
+    </div>
```

Comment:
Kindly notice that your social icons should be wrapped inside an `ul` element. This will improve the accessibility of your webpage and allows anyone to contact/check your social profiles with ease 😄. You should do it by creating an [ul](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul) and each `li` inside it should contain an `a` tag linking to your profile.  

Here is my tip for you :wink::  
```html  
<ul>  
  <li><a href="your social media URL">Social media icon</a></li>  
  <li><a href="another social media URL">Social media icon</a></li>  
  <li><a href="another social media URL">Social media icon</a></li> 
</ul>  
```